Lib9c/Action/TransferAsset2.cs

```diff
@@ -14,10 +14,11 @@ namespace Nekoyume.Action
 {
     /// <summary>
     /// Hard forked at https://github.com/planetarium/lib9c/pull/636
-    /// Updated at https://github.com/planetarium/lib9c/pull/957
+    /// Updated at https://github.com/planetarium/lib9c/pull/1412
     /// </summary>
     [Serializable]
     [ActionType("transfer_asset2")]
+    // Do not add `ActionObsolete` attribute while the Onboarding-portal uses this feature.
```

Comment:
Can you move this comment above of any attributes? In summary also OK I think.

Reply:
OK I understood. thanks~